### DISCUSSION TITLE
How to I can add hotjar code?

### DISCUSSION THREAD

**Discussion by SuperService** [nuxt/nuxt#17154] [Questions]
I updated my project with Nuxt3 v3.0.0-rc.5.
but I am getting issues
what is right way to add hotjar code on Nuxt3 Typescript?

**Comment by TheAlexLichter** [comment]
Can you provide the Hotjar code and/or an example of what you tried?

**Reply by SuperService** [reply]
I tried follow below
 yarn add @hotjar/browser

import Hotjar from '@hotjar/browser';
const siteId = 3081703;
const hotjarVersion = 6;
Hotjar.init(siteId, hotjarVersion);

500 error
Hotjar.init not function

I think that this problem is typescript problem 👀

**Reply by SuperService** [reply]
why no response?
